about.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">
+    <link rel="icon" href="images/fav-icon.ico">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="ham-menu ham-gone">
+        <a href="#" id="ham-close"><img class="ham-menu-close" src="images/pop-close.svg" alt=""></a>
+        <ul>
+            <li><a class="ham-links" href="index.html"> Home</a></li>
+            <li><a class="ham-links" href="about.html"> About</a></li>
+            <li class="ham-links">Program</li>
+        </ul>
+    </div>
+    <section class="about-main">
+        <nav>
+            <a href="#" id="ham"><img class="hamburger" src="images/hamburger-menu.png" alt=""></a>
+            <div class="desk-nav">
+                <div class="fixed-top">
+                    <ul class="fixed-ul">
+                        <li><i class="fa fa-facebook" aria-hidden="true"></i></li>
+                        <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                        <li>English</li>
+                        <li>My Page</li>
+                        <li>Logout</li>
```

Comment:
- [ ] As pointed out by the previous reviewer, please kindly ensure that the menu items(facebook, twitter, English, My page, Logout) use the anchor tag `<a></a>`